src/Microsoft.DotNet.Cli.Utils/HashHelper.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class HashHelper
+    {
+        public static string sha256_hash(string value)
+        {
+            StringBuilder Sb = new StringBuilder();
```

Comment:
Hashed arguments are still being debated.  However, this change will be made if it stays.